test/unit/app/browser/reducers/shieldsReducerTest.js

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/* global describe, it, before, after, afterEach */
+const sinon = require('sinon')
+const Immutable = require('immutable')
+const assert = require('assert')
+const appActions = require('../../../../../js/actions/appActions')
+const appConstants = require('../../../../../js/constants/appConstants')
+require('../../../braveUnit')
+
+describe('APP_TOGGLE_SHIELDS', function () {
```

Comment:
can you please wrap this tests with this code?

`describe('shieldsReducer unit tests', function () { ... }`